reactExercise/solutions/zihe/ex6/src/constants.js

```diff
@@ -0,0 +1,2 @@
+export const FRUITTIMER = 3
```

Comment:
可以用下划线来分隔全大写的constant variable name
```suggestion
export const FRUIT_TIMER = 3
```